tests/activityDetection.test.js

```diff
@@ -1,20 +1,20 @@
 // const activityDetection = require('../js/activityDetection/activityDetection.js');
 // const electron = require('./__mocks__/electron');
 
-xdescribe('Tests for Activity Detection', function() {
+xdescribe('Tests for Activity Detection', function () {
```

Comment:
why add space here?  this is not part of our linting rules???

also, other changes not related to the PR should be done separately.  thanks.